src/exceptions/SkillsException.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+import player.Player;
+
+public class SkillsException extends Exception {
```

Comment:
Convert to unchecked exception

```suggestion
public class SkillsException extends RuntimeException {
```